recipes/neurofire/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.1.0" %}
+{% set sha256 = "9f23eb77529fc34520c1a79022833f024cd4acb52ba406d08dc4d191372a7343" %}
+
```

Comment:
Please add `{% set name = "neurofire" %}` here (sorry I wasn't more specific about this earlier).

Reply:
Oh, and I thought there was some magic infering the name ;).
Anyway, I added the name variable now.

Reply:
Ah, no, it is a variable we define - I am just so used of using them all the time :)